app/javascript/app/utils/ms-formatter.js

```diff
@@ -14,5 +14,5 @@ export default (milliseconds) => {
   const displayMinutes = formatMinutes(minutes);
   const displaySeconds = formatSeconds(remainingSeconds);
 
-  return `${displayMinutes}:${displaySeconds}`;
+  return `00:${displayMinutes}:${displaySeconds}`;
```

Comment:
Is this change required so that screen readers read it in a particular time format? Alternatively, I wonder if we could just spell it out as the (translated) words, e.g. "15 minutes" or "14 minutes and 59 seconds".

Reply:
This approach makes it more machine readable so the screen reader can do more of the work for us. When it was `0:xx` the screen reader would say `24 xx seconds` and I have no idea why.  Instead of a span tag, I'm going with the time tag, and in that number format xx:xx:xx screen readers read it correctly, with native language support (i believe). If we wanted text there, it gets more complicated, we have to deal with plurals of `minutes` and `seconds` in our translated languages as well.

Reply:
That makes sense to me from a screen-reader, machine-readable point-of-view. I guess what I'm wondering is if we're okay with the usability impact this might have on the _visible_ text? In other words, does it make the text harder to read when it shows in the `HH:MM:SS` format? And if so, should we make this format apply only to the screen-reader text?

![image](https://user-images.githubusercontent.com/1779930/152827179-05b4bc77-14d4-4bad-ab25-9b62a1e10106.png)

Reply:
This unfortunately is similar to the current behavior where it's `XX:XX` format after the initial message. But as long as I'm here, let's change that to something more useful.